WordPress/WordPressTest/FeatureFlagTest.swift

```diff
@@ -25,8 +25,6 @@ class FeatureFlagTest: XCTestCase {
     func testEnsureDisabledFeaturesInProduction() {
         Build.withCurrent(.AppStore) {
             expect(FeatureFlag.People.enabled).to(beFalse())
-            expect(FeatureFlag.Plans.enabled).to(beFalse())
```

Comment:
I added them as part of my commit to fix the tests, as these ones were also failing. Want me to revert?